pages/agent/v3/windows.md.erb

```diff
@@ -4,7 +4,15 @@ The Buildkite Agent can be installed on both 64bit and 32bit editions of Windows
 
 <%= toc %>
 
-## Installing with Powershell
+## Automated install with Powershell
+
+You'll need to run the follow commands within Powershell with Administrative Privileges. You can do this (in Windows 8) by:
```

Comment:
'following' rather than 'follow'?

Reply:
And it might be worth clarifying a little at the end with 'To start a Powershell session as an Administrator (in Windows 8):'